Makefile

```diff
@@ -42,12 +43,17 @@ rapidwright:
 	cd third_party/RapidWright && make
 
 install_fasm2bels:
-	git submodule init
-	git submodule update
+	git submodule init third_party/fasm2bels
+	git submodule update third_party/fasm2bels
 	cd third_party/fasm2bels && make env
 	$(IN_ENV) cd third_party/fasm2bels && make build
 	cd third_party/fasm2bels && make test-py
 
+install_wafove:
+	cd third_party/WaFoVe && git pull origin master
```

Comment:
We should not run `git pull origin master`, since one key aspect of submodules is that they are tied to a particular commit.  This enables reproducible builds.  If we want to update bfasst, we can do that separately from updating wafove.
```suggestion

```